skgstat/interfaces/gstools.py

```diff
@@ -1,46 +1,87 @@
-try:
-    import gstools
-    GSTOOLS_AVAILABLE = True
-except ImportError:
-    GSTOOLS_AVAILABLE = False
+"""GSTools Interface."""
 import numpy as np
 
 
-def __check_gstools_available():  # pragma: no cover
-    if not GSTOOLS_AVAILABLE:
-        print('The gstools interface needs gstools installed.')
-        print("Run 'pip install gstools' to install it.")
-        return False
-    return True
+def stable_rescale(describe):
+    """Get GSTools rescale parameter from sk-gstat stable model description."""
+    return np.power(3, 1 / describe["shape"])
 
 
-def gstools_cov_model(variogram, **kwargs):
-    """GSTools Interface
+def matern_rescale(describe):
+    """Get GSTools rescale parameter from sk-gstat matern model description."""
+    if 0.5 < describe["smoothness"] < 10.0:
+        return 4.0
+    return 6.0
```

Comment:
This is a reminder to myself: I need to change the Matern model and always use 6.0 here.